xmltodict.py

```diff
@@ -326,6 +327,10 @@ def parse(xml_input, encoding=None, expat=expat, process_namespaces=False,
             parser.ExternalEntityRefHandler = lambda *x: 1
     if hasattr(xml_input, 'read'):
         parser.ParseFile(xml_input)
+    elif isinstance(xml_input, GeneratorType):
```

Comment:
Can you use [`inspect.isgenerator()`](https://docs.python.org/3/library/inspect.html#inspect.isgenerator) instead?

Reply:
Sure think. I think this is what you meant.